lib/resty/waf.lua

```diff
@@ -509,14 +513,63 @@ end
 function _M.new(self)
 	-- we need a separate copy of this table since we will
 	-- potentially override values with set_option
-	local t = util.table_copy(default_opts)
 
-	t.transaction_id = random.random_bytes(10)
+	local t = {
```

Comment:
we build and return this metatable every time we call `new`, but this all gets overwritten from `_load` inside `exec`. maybe we ought to just try-load here?